R/code_lineage.R

```diff
@@ -23,14 +23,14 @@ code_lineage <- function(title = NULL, database = NULL) {
     vars <- unlist(purrr::map(database, names))
     if (any("Text" == vars)) { # Find text variable in database, if available
       txt <- unname(unlist(purrr::map(database, "Text")))
-      txt <- retrieve_clauses(standardise_treaty_text(txt), "preamble")
+      txt <- read_clauses(standardise_treaty_text(txt), "preamble")
     }
   }
   # code entity and actions for titles
   title <- stringi::stri_trans_general(title, id = "Latin-ASCII")
   entity <- code_entity(title)
   domain <- code_domain(title)
-  parties <- code_parties(title)
+  parties <- code_states(title)
   # Get entity and actions from preamble if missing from title
   if (exists("txt")) {
     entity <- ifelse(is.na(entity), code_entity(txt), entity)
```

Comment:
Not sure where to put this but I think `code_entity()` and the `entity::location_entity()` that it depends on does not seem to be working well

Reply:
Thank you Jael! I will look into this